components/externalLinks.tsx

```diff
@@ -6,7 +6,8 @@ export default function ExternalLinks() {
     const { links } = document;
     for (let i = 0, linksLength = links.length; i < linksLength; i += 1) {
       if (links[i].hostname !== window.location.hostname) {
-        links[i].target = '_blank';
+        // these 2 add errors. Can't find a fix
```

Comment:
These only add errors in dev, not in prod. I think it may be something to do with the ISR in dev.